index.html

```diff
@@ -7,7 +7,7 @@ <h1 class="page-heading">EIPs
   <a href="https://gitter.im/ethereum/EIPs?utm_source=badge&amp;utm_medium=badge&amp;utm_campaign=pr-badge"><img src="https://badges.gitter.im/Join%20Chat.svg" alt="Gitter"></a>
   <a href="/last-call.xml"><img src="https://img.shields.io/badge/rss-Last Calls-red.svg" alt="RSS"></a>
 </h1>
-<p>Ethereum Improvement Proposals (EIPs) describe standards for the Ethereum platform, including core protocol specifications, client APIs, and contract standards.</p>
+<p>Ethereum Improvement Proposals (EIPs) describe standards for the Ethereum platform, including core protocol specifications, client APIs, and contract standards. Hardforks are discussed separately in the <a targe="_blank" href="https://github.com/ethereum/eth1.0-specs/">Eth1.0 Specifications</a> project.</p>
```

Comment:
the target for the <a> tag was missing a t
```suggestion
<p>Ethereum Improvement Proposals (EIPs) describe standards for the Ethereum platform, including core protocol specifications, client APIs, and contract standards. Hardforks are discussed separately in the <a target="_blank" href="https://github.com/ethereum/eth1.0-specs/">Eth1.0 Specifications</a> project.</p>
```